cpu/native/stdio_native/stdio_native.c

```diff
@@ -28,12 +28,20 @@ void stdio_init(void)
 
 ssize_t stdio_read(void* buffer, size_t max_len)
 {
-    return real_read(STDIN_FILENO, buffer, max_len);
+    ssize_t res;
+
+    res = real_read(STDIN_FILENO, buffer, max_len);
+
+    return res;
```

Comment:
Huh? Is this really improving readability?

Reply:
nope, this is just a leftover from 43b9f4f49cb923b9fc0db8ce9dab9dcf694e914f